src/tranquilo/options.py

```diff
@@ -111,20 +129,20 @@ class RadiusOptions(NamedTuple):
 
 
 class AcceptanceOptions(NamedTuple):
-    confidence_level: float = 0.8
-    power_level: float = 0.8
+    confidence_level: float = 0.95
+    power_level: float = 0.95
     n_initial: int = 5
-    n_min: int = 5
-    n_max: int = 100
+    n_min: int = 4
+    n_max: int = 50
```

Comment:
This should influence the noisy and non-noisy cases, have you checked whether the non-noisy benchmark results changed?